docs/user/tutorials/_sidebar.md

```diff
@@ -0,0 +1,19 @@
+* [Home](/)
+* [API Gateway module](/api-gateway/user/README.md)
```

Comment:
Except for the Home link, all the following links are broken.

Reply:
They should work when I add the /api-gateway/ alias